gremlin-go/example/go.mod

```diff
@@ -22,8 +22,9 @@ go 1.20
 require github.com/apache/tinkerpop/gremlin-go/v3 v3.6.1-rc2
```

Comment:
It would be better not to use a release candidate version of gremlin-go. Should likely be 3.6.5 or 3.7.0